semanticdb/metacp/src/main/scala/scala/meta/internal/javacp/Javacp.scala

```diff
@@ -92,9 +70,9 @@ object Javacp {
           addInfo(
             pkgSymbol,
             k.PACKAGE,
-            o.ACC_PUBLIC,
             pkgName,
             None,
+            o.ACC_PUBLIC,
```

Comment:
NB. I'm not sure if "Some(PUBLIC)" is the right accessibility for packages. Maybe it should be None instead. Doesn't matter for me.

Reply:
We already have a related ticket: https://github.com/scalameta/scalameta/issues/1325. Let's follow up there.